packages/forma-36-react-components/src/components/Table/TableRow/TableRow.css

```diff
@@ -6,5 +6,5 @@ tbody .TableRow:hover {
 
 .TableRow--selected {
   background-color: var(--color-blue-lightest);
-  box-shadow: inset -2px 0 0 var(--color-blue-mid);
+  border-right: 2px solid var(--color-blue-mid);
```

Comment:
@mshaaban0 I can't check out this branch because of the name 🙃 Did you check if this makes the content move around?

Reply:
<img width="1045" alt="image" src="https://user-images.githubusercontent.com/6163988/104594206-1e06fe00-5671-11eb-9110-5ea86e119a1a.png">

It looks alright, sorry about the branch, I think you can checkout `"` i.e. `git checkout 'fix(F36-476)/table'`

Reply:
I got it working. Check this out:

<img width="557" alt="Screenshot 2021-01-14 at 14 04 32" src="https://user-images.githubusercontent.com/219017/104594489-8229c200-5671-11eb-8299-e4018a2d5efe.png">

Try `box-sizing: border-box;`

Reply:
Can you point out what is the issue? I tried `box-sizing` on `tr` and couldn't notice any difference :)

Reply:
The border on the right extends outside the table width. But yes, now that you mention it `box-sizing` obviously won't work on `tr` 🙁

Reply:
yes you're right, it could causes a horizontal scroll as well